install-panel.sh

```diff
@@ -690,23 +703,22 @@ function letsencrypt {
     if [[ "$CONFIGURE_SSL" =~ [Yy] ]]; then
       ASSUME_SSL=true
       CONFIGURE_LETSENCRYPT=false
+      configure_nginx
     else
       ASSUME_SSL=false
       CONFIGURE_LETSENCRYPT=false
     fi
-  else 
-    systemctl restart nginx
   fi
 }
 
 #######################################
 ## WEBSERVER CONFIGURATION FUNCTIONS ##
 #######################################
 
-function configure_nginx {
+configure_nginx() {
   echo "* Configuring nginx .."
 
-  if [ "$ASSUME_SSL" == true ]; then
+  if "$ASSUME_SSL" && ! "$CONFIGURE_LETSENCRYPT"; then
```

Comment:
Why was the `[`, `]` removed?

Reply:
Because with them it didn't work right.

Reply:
I am a little unsure of what consequences not using square brackets has when it comes to functionality. As far as I know, the square bracket in bash is a shorthand notation for performing a conditional test. When using square brackets, bash "tests" the expression. By not using them, you are essentially testing the variable as an expression directly. In this case, `ASSUME_SSL` and `CONFIGURE_LETSENCRYPT` should not be anything else than booleans which leads to the same result since `true` has exit-code `0` when used as an expression and `false` has exit-code `1` when used as an expression.

Reply:
Ok, I will test once again.

Reply:
Ok I played around a bit,
```bash
#!/bin/bash

i=false
k=true

if $i && $k; then
  echo "Both true"
elif $i && ! $k; then
  echo "k not true"
elif ! $i && $k; then
  echo "i not true"
fi
```

And it works without any problem.

Reply:
Do you have a better solution (that involves `[ ]` )? I couldn't get it to work with `[ ]`. Also be sure that it is compliant with shellcheck.

Reply:
I've updated your PR. You think it looks OK? So using `!` basically means "not" whereas the variable is a boolean, true or false. Also, you were using `"` which actually makes the booleans into "strings" so that may have caused problems :slightly_smiling_face: You can rewrite whatever I wrote now but a bit more compact but this works as well!